### PR TITLE
Update metering report documentation

### PR DESCRIPTION
**Pull request by WeirdMachine**
Update the documentation for reporting and concepts to the new version.

It does not update the Dashboard controls as there are some more changes to come.

### CODE REVIEW

File: content/kubermatic/master/tutorials-howtos/metering/_index.en.md [hunk 1]
```diff
@@ -95,17 +99,35 @@ kind: Seed
...
-    # exporting it to s3 bucket. Default value is kubermatic-fast.
-    storageClassName: kubermatic-fast
+    # StorageClassName is the name of the storage class that the metering prometheus instance uses to store metric data for reporting.
+    storageClassName: ""
     # StorageSize is the size of the storage class. Default value is 100Gi.
     storageSize: 100Gi
+    reports:
+      weekly:
+        interval: 7
+        schedule: 0 1 * * 6
```
Comment (pkprzekwas) [inline content/kubermatic/master/tutorials-howtos/metering/_index.en.md:109]: We could add retention to the example:
```suggestion
        interval: 7
        retention: 90
        schedule: 0 1 * * 6
```

Comment (WeirdMachine) [reply content/kubermatic/master/tutorials-howtos/metering/_index.en.md:122]: I added the section about report configuration via the dashboard. IMO this should be ok. Types are kind of self explaining.

File: content/kubermatic/master/tutorials-howtos/metering/_index.en.md [hunk 2]
```diff
@@ -63,9 +67,23 @@ Click on **Configure Metering**, switch on **Enable Metering** and change the co
...
+A schedule consist of four different values to set: 
+
+- `Schedule Name`
+  - Name of the Schedule. This is also used as a folder name to store generated reports.
+- `Report retention`
+  - Number of days each report is saved, leave the field empty to store reports forever. This will set a retention period at the s3 Backend. 
+- `Report scope`
+  - Number of days captured in each report.
+- `Cron Expression`
+  -  Cron expression that describes how often a report should be created.
```
Comment (pkprzekwas) [inline content/kubermatic/master/tutorials-howtos/metering/_index.en.md:84]: nit: needless whitespace
```suggestion
  - Cron expression that describes how often a report should be created.
```

**Pull request merged**